internal/bucket/lifecycle/lifecycle.go

```diff
@@ -235,9 +235,9 @@ func (lc Lifecycle) Validate() error {
 	return nil
 }
 
-// FilterActionableRules returns the rules actions that need to be executed
-// after evaluating prefix/tag filtering
-func (lc Lifecycle) FilterActionableRules(obj ObjectOpts) []Rule {
+// FilterRulesByStatusPrefixAndTags returns the rules actions that need to be executed
```

Comment:
I think we can avoid exporting this method, it is simply a helper function for other higher-level exported methods.